sdk/identity/identity/review/identity.api.md

```diff
@@ -88,6 +88,7 @@ export class AzureCliCredential implements TokenCredential {
 
 // @public
 export interface AzureCliCredentialOptions extends MultiTenantTokenCredentialOptions {
+    processTimeout?: number;
```

Comment:
I think we usually suffix the units in the name, so `processTimeoutInMs` ?

Reply:
updated